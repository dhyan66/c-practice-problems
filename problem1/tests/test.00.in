new 30
set 1
set 3
set 5
set 10
set 11
end
