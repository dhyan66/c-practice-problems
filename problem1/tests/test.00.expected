00 00 00 00 
02 00 00 00 
0a 00 00 00 
2a 00 00 00 
2a 04 00 00 
2a 0c 00 00 
